packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, arrayOf } from 'prop-types';
+import { EasternArabic, WesternArabic } from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadLink } from './item';
+
+const MostReadItemColumns = {
+  columns: {
+    group0: 5,
+    group1: 5,
+    group2: 5,
+    group3: 5,
+    group4: 5,
+    group5: 5,
+  },
+};
+
+const MostReadRankColumns = {
+  item: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 1,
+    group3: 1,
+    group4: 1,
+    group5: 1,
+  },
+};
+
+const MostReadLinkColumns = {
+  item: true,
+  columns: {
+    group0: 4,
+    group1: 4,
+    group2: 4,
+    group3: 4,
+    group4: 4,
+    group5: 4,
+  },
+};
+
+const MostReadListColumns = {
+  enableGelGutters: true,
+  enableGelMargins: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 2,
+    group3: 2,
+    group4: 2,
+    group5: 5,
+  },
+};
+
+const serviceNumerals = {
+  news: WesternArabic,
+  arabic: EasternArabic,
```

Comment:
This list should have the other numerals in it, too. For example: 
```
bengali: Bengali,
burmese: Burmese,
```
Ideally, we should have all of the other services defined too - we can move these out into Simorgh config later on. 

e.g. to have a full list of the services:
```
afaanoromo: WesternArabic,
...
zhongwen: WesternArabic,
```

Reply:
Resolved - thanks to you!